src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int radius;
+    private Random random = new Random();
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void randomizeProperty() {
+        radius = random.nextInt(MAXIMUM_SIZE);
+    }
```

Comment:
The `randomizeProperty` method should not be responsible for generating random values. Instead, consider passing the random value for the radius through the constructor or a setter method.